deployment/arm/cosmosdb.json

```diff
@@ -0,0 +1,85 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"primaryRegion":{
+			"type":"string",
+            "defaultValue": "westus2",
```

Comment:
Let's fix up the white space to be consistent so it is easier to read.

Reply:
Strange but this is all aligned on my local copy. I`ll figure this out.